### PR TITLE
Arachne 1404

### PR DESCRIPTION
**Pull request by MikMironov**

### CODE REVIEW

File: src/modules/InsightsLibrary/components/ViewEdit/ModalSettings/presenter.jsx [hunk 1]
```diff
@@ -42,7 +43,8 @@ function ModalAccessSettings(props) {
           <div {...classes('setting')}>
             <div {...classes('setting-title')}>Publish state</div>
             <Tabs
-              {...classes('setting-value')}
+              {...classes({element: 'setting-value',
```
Comment (agackovka) [inline src/modules/InsightsLibrary/components/ViewEdit/ModalSettings/presenter.jsx]: Don't forget about formatting (spaces near curly brackets)

File: src/modules/InsightsLibrary/components/ViewEdit/ModalSettings/container.js [hunk 2]
```diff
@@ -42,6 +43,7 @@ export default class ModalSettingsBuilder {
     return {
       publishState: get(state, 'insightsLibrary.insights.data.publishState', 'DRAFT', 'String'),
       insightId: get(state, 'insightsLibrary.insights.data.id'),
+      isPublishAvl: get(state, 'insightsLibrary.insights.data.study.status.availableActions', []).includes(studyActions.publishPaper),
```
Comment (pavgra) [inline src/modules/InsightsLibrary/components/ViewEdit/ModalSettings/container.js]: Let's use same names for same things. So call it "canPublishPaper" is it is done below

File: src/modules/InsightsLibrary/components/ViewEdit/ModalSettings/presenter.jsx (hunk 1 above)
Comment (pavgra) [reply src/modules/InsightsLibrary/components/ViewEdit/ModalSettings/presenter.jsx]: this is actual, when we do import or parameters pass. here everything is ok

Comment (agackovka) [reply src/modules/InsightsLibrary/components/ViewEdit/ModalSettings/presenter.jsx]: try to find 'classes({ ' across the project, and after try 'classes\(\{\S'

File: src/modules/InsightsLibrary/components/ViewEdit/ModalSettings/presenter.jsx [hunk 3]
```diff
@@ -42,7 +43,8 @@ function ModalAccessSettings(props) {
           <div {...classes('setting')}>
             <div {...classes('setting-title')}>Publish state</div>
             <Tabs
-              {...classes('setting-value')}
+              {...classes({ element: 'setting-value',
```
Comment (pavgra) [inline src/modules/InsightsLibrary/components/ViewEdit/ModalSettings/presenter.jsx]: Pls, rebuild it in such way, that would remove code duplication in Enterprise

**Pull request merged**